tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -583,184 +610,194 @@ class TfGraphControls extends LegacyElementMixin(PolymerElement) {
         </paper-radio-group>
         <span class="spacer"></span>
       </div>
-      <div>
-        <template is="dom-if" if="[[_isGradientColoring(stats, colorBy)]]">
-          <svg width="140" height="20" style="margin: 0 5px" class="color-text">
-            <defs>
-              <linearGradient
-                id="linearGradient"
-                x1="0%"
-                y1="0%"
-                x2="100%"
-                y2="0%"
-              >
-                <stop
-                  class="start"
-                  offset="0%"
-                  stop-color$="[[_currentGradientParams.startColor]]"
-                ></stop>
-                <stop
-                  class="end"
-                  offset="100%"
-                  stop-color$="[[_currentGradientParams.endColor]]"
-                ></stop>
-              </linearGradient>
-            </defs>
-            <rect
-              x="0"
-              y="0"
-              width="135"
+    </div>
+    <div class="legend-holder">
+      <div class="legend-toolbar" on-click="_toggleLegendOpen">
```

Comment:
Please do not make div clickable. Use `<button>` or something more naturally interactable clickable.

Reply:
Done as `<button>`.